Week3/homework/src/actions/updateTodoList.js

```diff
@@ -0,0 +1,30 @@
+'use-strict';
+
+const connection = require('../databaseConfig');
+
+const updateTodoList = (req, res) => {
+  if (req.body.todoListName === '' || req.body.todoListName === null) {
```

Comment:
Non-blocking:

I see here and there you do validation as the first step in your code. This is good. And it's a repetitive action.
There is DRY rule: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

I'm saying so because of your colleagues will tell you this as well. Better be prepared.